doc/usermanual/po/it.po

```diff
@@ -4,9 +4,17 @@
 # Maurizio Paglia <mpaglia0@gmail.com>, 2016, 2017, 2018, 2019.
 msgid ""
 msgstr ""
-"Project-Id-Version: darktable 2.4\n"
+"Project-Id-Version: darktable 2.6\n"
 "POT-Creation-Date: 2019-03-14 20:49+0100\n"
+<<<<<<< HEAD
```

Comment:
non resolved conflicts committed !

Reply:
I did an hard work to "rebase" the repository. This is an error that yesterday I tried to solve uploading the final version. Sorry.